### PR TITLE
6.1 links frontend GET decks api to backend

### PR DESCRIPTION
**Pull request by danzo3000**
links api utils to the backend and renders the data successfully on the app

### CODE REVIEW

File: components/Decks.jsx [hunk 1]
```diff
@@ -2,22 +2,22 @@ import React, { useEffect } from "react";
...
-  const testDecks = [
-    { name: "JS", description: "Testtesttest", id: 1 },
-    { name: "Python", description: "Testtesttest", id: 2 },
-    { name: "HTML", description: "Testtesttest", id: 3 },
-  ];
   const [currentDecks, setCurrentDecks] = useState([]);
 
-  useEffect(() => {}, []);
+  useEffect(() => {
+    getDecks().then((decks) => {
```
Comment (RogerioSoares96) [inline components/Decks.jsx:11]: Maybe add a loading state ?

File: components/Decks.jsx [hunk 2]
```diff
@@ -41,11 +41,11 @@ const Decks = ({ navigation }) => {
         </Pressable>
       </View>
       <FlatList
```
Comment (RogerioSoares96) [inline components/Decks.jsx:43]: Well used the flatlist ! going to check this locally on my phone :)

File: utils/api.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import axios from "axios";
 
-const api = axios.create({ baseURL: "" });
+const api = axios.create({ baseURL: "https://dev-cards.onrender.com/api" });
 
 export const postDeck = (name, description) => {
   return api
```
Comment (RogerioSoares96) [inline utils/api.js:6]: Simple and concise, well done !

File: components/Decks.jsx [hunk 4]
```diff
@@ -2,22 +2,22 @@ import React, { useEffect } from "react";
...
-    { name: "JS", description: "Testtesttest", id: 1 },
-    { name: "Python", description: "Testtesttest", id: 2 },
-    { name: "HTML", description: "Testtesttest", id: 3 },
-  ];
   const [currentDecks, setCurrentDecks] = useState([]);
 
-  useEffect(() => {}, []);
+  useEffect(() => {
+    getDecks().then((decks) => {
+      setCurrentDecks(decks);
```
Comment (RogerioSoares96) [inline components/Decks.jsx:12]: Also Error state ?

File: utils/api.js (hunk 3 above)
Comment (RogerioSoares96) [inline utils/api.js:6]: On line 19 of this file, _(data)_ is not in the correct format use : ({data})

Comment (RogerioSoares96) [reply utils/api.js:6]: same for line 8

File: components/Decks.jsx (hunk 2 above)
Comment (RogerioSoares96) [reply components/Decks.jsx:43]: @danzo3000 will link up with you to get more info on this :)

**Comment by GeorgeTsez** [comment]
looks great! keep up the good work

**Pull request merged**